### PR TITLE
LPS-199974 New layout for Sites

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/frontend/taglib/clay/servlet/taglib/SelectSiteInitializerVerticalCard.java [hunk 1]
```diff
@@ -36,6 +37,12 @@ public SelectSiteInitializerVerticalCard(
 		_httpServletRequest = PortalUtil.getHttpServletRequest(renderRequest);
 	}
 
+	@Override
+	public String getAriaLabel() {
+		return LanguageUtil.get(_httpServletRequest, "select-template") + ": " +
+			_siteInitializerItem.getName();
```
Comment (victorg1991) [inline modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/frontend/taglib/clay/servlet/taglib/SelectSiteInitializerVerticalCard.java:43]: can we add a language key specific to this? 😢

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:test:stable

**Comment by victorg1991** [comment]
ci:test:stable

**Comment by victorg1991** [comment]
ci:forward:force

**Pull request closed without merging**